src/domain/signupGroup/eventInfo/EventTimes.tsx

```diff
@@ -0,0 +1,60 @@
+import { useTranslation } from 'next-i18next';
+import { FC, useState } from 'react';
+
+import Accordion from '../../../common/components/accordion/Accordion';
+import LoadingSpinner from '../../../common/components/loadingSpinner/LoadingSpinner';
+import useLocale from '../../../hooks/useLocale';
+import useEventWithSubEventsData from '../../event/hooks/useEventWithSubEventsData';
+import { Event } from '../../event/types';
+import { getEventFields } from '../../event/utils';
+
+import DateText from './DateText';
+import styles from './eventInfo.module.scss';
+
+export type EventTimesProps = {
+  event: Event;
+};
+
+const EventTimes: FC<EventTimesProps> = ({ event }) => {
+  const { t } = useTranslation('signup');
+  const [open, setOpen] = useState(false);
+  const locale = useLocale();
+
+  const toggle = () => {
+    setOpen((s) => !s);
+  };
+
+  const { eventWithSubEvents, isLoading: isLoadingEvent } =
+    useEventWithSubEventsData({
+      id: event.id,
+      superEventType: event.super_event_type,
+    });
+
+  if (!event.super_event_type) {
+    return null;
+  }
+
+  return (
+    <Accordion
+      className={styles.eventTimesAccordion}
+      open={open}
+      onClick={toggle}
+      toggleButtonLabel={t('signup:eventTimes')}
+    >
+      <LoadingSpinner isLoading={isLoadingEvent}>
+        <ul>
+          {eventWithSubEvents?.sub_events.map((item) => {
```

Comment:
To ensure an intuitive ordering: maybe these could be sorted firstly by `start_time` so that sub-events with NULL values are at the end of the list?

Reply:
Good point. Add sorting for the sub-events